tests/e2e/cucumber/features/smoke/internalLink.ocis.feature

```diff
@@ -0,0 +1,34 @@
+Feature: internal link share
+
+  Scenario: share a link with internal role
+    Given "Admin" creates following users
+      | id    |
+      | Alice |
+      | Brian |
+    And "Alice" creates the following folder in personal space using API
+      | name     |
+      | myfolder |
+    And "Alice" shares the following resource using API
+      | resource | recipient | type | role   |
+      | myfolder | Brian     | user | editor |
+    And "Alice" logs in
+    And "Alice" opens the "files" app
+    And "Alice" creates a public link for the resource "myfolder" using the sidebar panel
+    When "Alice" edits the public link named "Link" of resource "myfolder" changing role to "internal"
+    And "Brian" opens the public link "Link"
+    And "Brian" logs in from the internal link
+    And "Brian" opens shared-with-me page from the internal link
+    And "Brian" accepts the following share
+      | name     |
+      | myfolder |
+    And "Brian" uploads the following resource
+      | resource   | to       |
+      | simple.pdf | myfolder |
+    And "Alice" updates following sharee role
+      | resource | recipient | type | role                    |
+      | myfolder | Brian     | user | custom_permissions:read |
+    And "Alice" logs out
+    Then "Brian" should not be able to reshare the following resource
+      | resource |
+      | myfolder |
+    And "Brian" logs out
```

Comment:
In my view, we should check whether we should able to write or not to.
Or it will be too much then step

Reply:
I'd change step `Then "Brian" should not be able to reshare the following resource`
to step like https://github.com/owncloud/web/blob/master/tests/e2e/cucumber/steps/ui/spaces.ts#L119 
because we use `Then "Brian" should not be able to reshare the following resource` in `reshare.feature`